template/web/src/components/SmartWallets/PaymasterClient.ts

```diff
@@ -1,7 +1,16 @@
-import { createPublicClient, http } from 'viem';
+import { ENTRYPOINT_ADDRESS_V06 } from 'permissionless';
+import { paymasterActionsEip7677 } from 'permissionless/experimental';
+import { createPublicClient, createClient, http } from 'viem';
 import { baseSepolia } from 'viem/chains';
 
+const paymasterService = process.env.NEXT_PUBLIC_PAYMASTER_URL ?? '';
+
 export const client = createPublicClient({
```

Comment:
As this is a general util can just live in `src/utils/client` and `src/utils/paymasterClient`

Reply:
I moved the paymasterClient to be under `@/utils/`